packages/sak-visittkort/src/components/VisittkortPanel.tsx

```diff
@@ -71,7 +71,10 @@ const VisittkortPanel: FunctionComponent<OwnProps> = ({
   }
 
   const soker = personopplysninger;
-  const annenPart = personopplysninger.annenPart ? personopplysninger : personopplysninger.annenPart;
+  const annenPart =
+    typeof personopplysninger.annenPart !== 'undefined' && personopplysninger.annenPart !== null
+      ? personopplysninger.annenPart
+      : null;
```

Comment:
Hvis null er en gyldig verdi på annenPart så trenger vi jo egentlig ikke sjekke for null? Ikke at dette endrer utfallet av sjekken, men blir vel ca 45 tegn mindre i koden 😄

Reply:
True true! Har fixat det nu :D